pkg/activator/net/lb_policy_test.go

```diff
@@ -158,6 +158,38 @@ func TestFirstAvailable(t *testing.T) {
 	})
 }
 
+func TestRoundRobin(t *testing.T) {
+	rrp := newRoundRobinPolicy()
+	podTrackers := makeTrackers(3, 1)
+	cb, pt := rrp(context.Background(), podTrackers)
+	t.Cleanup(cb)
+	if got, want := pt, podTrackers[0]; got != want {
+		t.Fatalf("Tracker = %v, want: %v", got, want)
+	}
+	cb, pt = rrp(context.Background(), podTrackers)
+	t.Cleanup(cb)
+	if got, want := pt, podTrackers[1]; got != want {
+		t.Fatalf("Tracker = %v, want: %v", got, want)
+	}
+	cb, pt = rrp(context.Background(), podTrackers)
+	if got, want := pt, podTrackers[2]; got != want {
+		t.Fatalf("Tracker = %v, want: %v", got, want)
+	}
+	_, pt = rrp(context.Background(), podTrackers)
+	t.Cleanup(cb)
+	if pt != nil {
+		t.Fatal("Wanted nil, got: ", pt)
+	}
```

Comment:
worth adding a test for properly wrapping around?

Reply:
I think it does, but I can do a more explicit one.

Reply:
I tried commenting out the `% l` and nothing failed, but I might (well) have goofed